README.md

```diff
@@ -1,179 +1,115 @@
+
+- Notifications in Home Assistant UI
 
   ![Notification](images/notification_error.png)
 
-* A [binary_sensor](#lovelace-card) you can use to trigger additional actions.
-  
-   ![Binary Sensor](images/binary_sensor.png)
+- A [binary_sensor](#lovelace-card) you can use to trigger additional actions.
 
-Redundancy is the foundation of reliability.  With local snapshots, Google Drive's backups, and two flavors of notification I think you're covered.
+  ![Binary Sensor](images/binary_sensor.png)
+
+Redundancy is the foundation of reliability. With local snapshots, Google Drive's backups, and two flavors of notification I think you're covered.
 
 ### How do I restore a snapshot?
-If you can still get to the addon's web-UI then can select "Actions" -> "Upload" from any snapshot to have it copied back into Home Assistant. If not, you'll need to start up a fresh installation of Hass.io and do the following:
-- On your fresh install of Hass.io:
-  *  [Install the add-on.](#installation)  Once linked with Google Drive, you should see the snapshots you created previously show up.  You should see a warning pop up about finding an "Existing snapshot folder" which is expected, you can just ignore it for now.
-  * Click "Actions" -> "Upload" on the snapshot you want to use which will upload the snapshot to Home Assistant directly from Google Drive.  Wait for the upload to finish.
-  * Click "Actions" -> "Restore" to be taken to the Hass.io restore page, or just navigate there through the Home Assistant interface ("Hass.io" -> "Snapshots").
-  * You'll see the snapshot you uploaded.  Click on it and select "Wipe & Restore".  Wait a while for it to complete (maybe a long while).  Congrats! you're back up and running.
 
-Note: You can also just copy a snapshots manually from Google Drive to your /backup folder using something like the Sambda addon.  I've found the steps above to be a little easier, since it works for any operating system, network setup, etc.
+If you can still get to the addon's web-UI then can select "Actions" -> "Upload" from any snapshot to have it copied back into Home Assistant. If not, you'll need to start up a fresh installation of Home Assistant and do the following:
+
+- On your fresh install of Home Assistant:
+  - [Install the add-on.](#installation) Once linked with Google Drive, you should see the snapshots you created previously show up. You should see a warning pop up about finding an "Existing snapshot folder" which is expected, you can just ignore it for now.
+  - Click "Actions" -> "Upload" on the snapshot you want to use which will upload the snapshot to Home Assistant directly from Google Drive. Wait for the upload to finish.
+  - Click "Actions" -> "Restore" to be taken to the Home Assistant restore page, or just navigate there through the Home Assistant interface ("Home Assistant" -> "Snapshots").
```

Comment:
"Home Assistant" on this line should be "Supervisor", since thats how it shows up in the UI now.

Reply:
```suggestion
  - Click "Actions" -> "Restore" to be taken to the Home Assistant restore page, or just navigate there through the Home Assistant interface ("Supervisor" -> "Snapshots").
```